sdk/keyvault/.azure-pipelines/pr.client.yml

```diff
@@ -0,0 +1,19 @@
+# DO NOT EDIT THIS FILE
+# This file is/may be generated automatically and any changes may be lost.
+
+
+pr:
```

Comment:
Is this pr section necessary? master is the default correct?

Reply:
I'm going to test to make sure that master is the default and that its not a wildcard, once I've checked into that I'll make the necessary modifications. The documentation is a little ambiguous.

Reply:
So the ```pr:``` statement is redundant in that PR validation is the default behaviour, but it is the default behavior on all branches. This may or may not be desirable. For the time being I am going to remove it as unnecessary but we should consider whether we want to force these pipelines to run when folks want to merge between branches other than master (not a bad thing on the surface but could be annoying in some peoples workflows).